src/backend/distributed/citus--6.1-17--6.1-18.sql

```diff
@@ -0,0 +1,22 @@
+/* citus--6.1-17--6.1-18.sql */
+
+SET search_path = 'pg_catalog';
+
+DROP FUNCTION IF EXISTS master_add_node(text, integer);
+
+CREATE FUNCTION master_add_node(nodename text,                                                
+                                nodeport integer,
+                                replicateReferenceTables boolean DEFAULT TRUE,                                             
```

Comment:
There is a question whether we would want to add a new parameter or use a GUC. The drawback of the former is that Cloud would require different behaviour for adding a node in 6.0 and 6.1.